File: Backend/Backend/Repositories/Models/Word.cs
using System;

namespace Backend.Repositories.Models
{
    public class Word
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Translation { get; set; }
        public string Grammar { get; set; }
        public string Story { get; set; }
        public DateTime CreatedTime { get; set; }
        public DateTime UpdatedTime { get; set; }

        public Word()
        {

        }

        public Word(string name, string translation, string grammar, string story)
        {
            Name = name;
            Translation = translation;
            Grammar = grammar;
            Story = story;
        }

        public Word(string id, string name, string translation, string grammar, string story)
        {
            Id = id;
            Name = name;
            Translation = translation;
            Grammar = grammar;
            Story = story;
        }
    }
}

File: backend/src/database/models/word.js
import Sequelize from 'sequelize'
import db from '../db'

const word = db.define('word', {
    id: {
        type: Sequelize.UUID,
        primaryKey: true
    },
    name: {
        type: Sequelize.STRING
    },
    translation: {
        type: Sequelize.STRING
    },
    grammar: {
        type: Sequelize.STRING
    },
    story: {
        type: Sequelize.STRING
    },
    createdAt: {
        type: Sequelize.DATE
    },
    updatedAt: {
        type: Sequelize.DATE
    },
    enabled: {
        type: Sequelize.BOOLEAN
    }
})

export default word
